include/proxysql_utils.h

```diff
@@ -8,6 +8,9 @@
 #include <vector>
 #include <sys/time.h>
 
+#ifdef __FreeBSD__
+#define	ETIME	ETIMEDOUT
```

Comment:
This is probably not the right way of dealing with the fact that `ETIME` is missing in FreeBSD , and probably all the timer functionality needs to be rewritten for compatibility.